app/src/main/res/values-fr/settings.xml

```diff
@@ -34,6 +34,15 @@
     <item>2</item>
   </string-array>
 
+  <string-array name="widget_width_types">
+    <item>default</item>
+    <item>Single stock in a row</item>
```

Comment:
can we get the correct french translations here

Reply:
Done